docling_core/types/doc/document.py

```diff
@@ -981,6 +1200,23 @@ class PageItem(BaseModel):
 class DoclingDocument(BaseModel):
     """DoclingDocument."""
 
+    HTML_DEFAULT_HEAD: str = r"""<head>
```

Comment:
make this `_HTML_DEFAULT_HEAD`, then we don't need any weird delete logic below

Reply:
Resolved.